### DISCUSSION TITLE
Deployment to netlify failed

### DISCUSSION THREAD

**Discussion by jeffreys-cat** [facebook/docusaurus#7897] [Q&A]
Here is my repo [https://github.com/colordove/doris-website](https://github.com/colordove/doris-website)，build success on my mac and GitHub Action.

![image](https://user-images.githubusercontent.com/11832969/182843177-12294450-2dfc-4329-98a2-ca9e6cb3f651.png)

But it failed on Netlify. I don't know what happened....

https://app.netlify.com/sites/apache-doris/deploys/62ebb1017145de0065a6b12d

![image](https://user-images.githubusercontent.com/11832969/182843424-737fefd8-b241-4e14-9ae4-a20c0b977c5e.png)

**Comment by slorber** [comment]
Already had this error, IMHO it's just temporary

Can you retry the build with the Netlify cache cleared?

**Reply by jeffreys-cat** [reply]
Thank you, It works on Vercel, I will clear the cache and try again on Netlify.
